src/ml/neural_net/mps_layer_instance_norm_data_loader.mm

```diff
@@ -54,11 +54,11 @@ - (instancetype) initWithParams:(NSString *)name
     
     _currentStyle = 0;  
     
-    _gamma_weights = (float *) malloc(numberFeatureChannels * styles * sizeof(float));
-    _beta_weights = (float *) malloc(numberFeatureChannels * styles * sizeof(float));
+    _gamma_weights = [NSMutableData dataWithLength:numberFeatureChannels * styles * sizeof(float)];
+    _beta_weights = [NSMutableData dataWithLength:numberFeatureChannels * styles * sizeof(float)];
 
-    memcpy(_gamma_weights, gammaWeights, numberFeatureChannels * styles * sizeof(float));
-    memcpy(_beta_weights, betaWeights, numberFeatureChannels * styles * sizeof(float));
+    memcpy(_gamma_weights.mutableBytes, gammaWeights, numberFeatureChannels * styles * sizeof(float));
```

Comment:
Hmm, instead of creating an empty NSMutableData and then filling it, you should be able to do something like
```
_gamma_weights = [NSMutableData dataWithBytes:gammaWeights length:numberFeatureChannels * styles * sizeof(float)];
```

Reply:
I should have looked at the api a bit closer. That seems to work!